### DISCUSSION TITLE
licence question

### DISCUSSION THREAD

**Discussion by mesqueeb** [millsp/ts-toolbelt#181] [Wiki]
#### Describe your question

Heya!

I use ts-toolbelt as a dependency on quite some small libraries of mine. I usually choose the MIT License and not think this over too much, I'm not really legally conscious about this. : O

But today I noticed that TS-Toolbelt has Apache 2.0 license.

Is that a problem if ts-toolbelt is a dependency of one of my MIT libraries?

Cheers!

#### Search tags, topics
#license

**Comment by millsp** [comment]
Since my code isn't bundled, I'd say that's ok. You can always give your projects a check on https://app.fossa.com/projects/git%2Bgithub.com%2Fpirix-gh%2Fts-toolbelt/refs/branch/master/be381423a520c890edcd78b894184d8f7dc52eab/preview

I have a block at the bottom of the README for this.

Cheers
